examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py

```diff
@@ -127,13 +117,13 @@
 # We can now zoom in on the region in the EUVI image:
 subeuvi = maps['EUVI'].submap(hpc_B[0], hpc_B[3])
 fig = plt.figure()
+plt.subplot(projection=subeuvi)
 subeuvi.plot()
 
 ###############################################################################
 # Putting them together:
-fig = plt.figure(figsize=(15, 5))
+fig = plt.figure()
```

Comment:
no don't take my figsize away!!
```suggestion
fig = plt.figure(figsize=(15, 5))
```